src/ngraph/ngraph_emitter.cc

```diff
@@ -1261,6 +1262,24 @@ void Emitter::CreateLayerOps() {
 
     return mul_op;
   };
+  ngraph_op_funcs_["SequenceReverse"] =
+      [this](const NodePtr& node) -> NgraphNodePtr {
+    auto data = op_map_[node->inputs_[0]];
+    bool use_sequence_length = get_default(node, "use_sequence_length", false);
+    int seq_axis = get_default(node, "axis", 0);
+    const int batch_axis = 1;
+    NgraphNodePtr sequence_length{nullptr};
+    if (use_sequence_length) {
+      sequence_length = op_map_[node->inputs_[1]];
+    } else {
+      const ngraph::Shape& data_shape = data->get_shape();
+      sequence_length = makeConstant(data->get_element_type(),
+                                     ngraph::Shape{data_shape.at(batch_axis)},
+                                     std::to_string(data_shape.at(seq_axis)));
```

Comment:
Would a standard op::Reverse make more sense in this case?

Reply:
Thanks, I didn't look at that op. New fix will use `ngraph::op::Revervse` by default.